plugins/admin/pkg/command/registry/remove.go

```diff
@@ -55,16 +56,24 @@ func NewRegistryRmCommand(p *pkg.AdminParams) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if p.Namespace == "" {
+				cmd.Print("No namespace specified, using 'default' namespace\n")
```

Comment:
Being verbose is good, but I think here we can omit the output, but I suggest to make `default` as the default value of namespace and ServiceAccount. And at the end it wil print the namespace and ServiceAcount name once done.

Reply:
Thanks, I have removed these logs and also the assertion in ut.